packages/amplify-util-uibuilder/src/commands/utils/codegenResources.ts

```diff
@@ -1,4 +1,5 @@
-import { GenericDataSchema, getGenericFromDataStore } from '@aws-amplify/codegen-ui';
+import { getGenericFromDataStore } from './codegen-ui';
```

Comment:
We don't need two mapping functions now. We can just map straight to the SDK type being used in StartCodegenJob

Reply:
`getGenericFromDataStore` returns the `genericDataSchema` parameter that is consumed by `mapGenericDataSchemaToCodegen`

Reply:
Do you mean to combine them into one function?

Reply:
one function that goes from introspection schema to codegen

Reply:
`getGenericFromDataStore` is a long function already. It doesnt look very clean if its combined with `mapGenericDataSchemaToCodegen`